modules/social_features/social_follow_user/social_follow_user.module

```diff
@@ -97,14 +97,20 @@ function social_follow_user_profile_view(
   EntityViewDisplayInterface $display,
   string $view_mode
 ): void {
-  if (
-    $entity->bundle() === 'profile' &&
-    $display->getComponent('social_follow_user') !== NULL
-  ) {
-    $build['social_follow_user'] = [
-      '#theme' => 'followers',
-      '#flaggable' => $entity,
-    ];
+  if ($entity->bundle() === 'profile') {
+
+    if ($display->getComponent('social_follow_user') !== NULL) {
+      $build['social_follow_user'] = [
+        '#theme' => 'followers',
+        '#flaggable' => $entity,
+      ];
+    }
+    if ($display->getComponent('social_following_user') !== NULL) {
+      $build['social_following_user'] = [
```

Comment:
Define the field in the "social_follow_user_entity_extra_field_info" hook and make the following changes:
```suggestion
    if ($display->getComponent('following') !== NULL) {
      $build['following'] = [
```